### PR TITLE
Math-magicians: Events

### PR DESCRIPTION
**Pull request by ntzwilly**
## Description:

- Add **calculate.js** and **operate.js** into a **logic/** directory
- Import the files in the Calculator component.
- Implement the event handlers.
- Test the app in the browser.

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+  if (operation === '÷') {
+    try {
+      return one.div(two).toString();
+    } catch (err) {
+      return "Can't divide by 0.";
+    }
+  }
```
Comment (Emmyn5600) [inline src/logic/operate.js:21]: - [x] Kindly, display the user-friendly message once a user divides a number by zero 

date-1

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi,

Good job so far! 👏🏼 @ntzwilly 

Descriptive commit messages :heavy_check_mark:
Well written PR :heavy_check_mark:

There are some issues that you still need to work on to go to the next project but you are almost there!

## STATUS: CHANGES REQUESTED :recycle:


### Changes requested.

- _Check the comment under the review_

 #### Optional suggestions
 
 - N/A
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/logic/operate.js (hunk 1 above)
Comment (ntzwilly) [reply src/logic/operate.js:21]: Hi @Emmyn5600 , Thank you for your feedback.

However, I think this issue is not part of the requirements. These files (**calculate.js** and **operate.js**) were provided to us. We only need to use their logic in order to implement the event handlers for the Calculator component. All math operations are working and the error message is the default, not my implementation.
error_message
Kindly have a second look at this as we have a limited number of code reviews.

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @ntzwilly, 

Everything looks good to me. Well Done.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**